src/urdf_parser_py/urdf.py

```diff
@@ -324,7 +324,7 @@ def __init__(self, name=None, visual=None, inertial=None, collision=None, origin
 		self.collision = collision
 		self.origin = origin
 
-xmlr.reflect(Link, params = [
+xmlr.reflect(Link, tag = 'link', params = [
```

Comment:
Same, There should be no spaces next to equal sign. Of course, nothing else in this module follows pep 8, but it'd be nice to start.